export/filter_mac.go

```diff
@@ -0,0 +1,24 @@
+package export
+
+import "github.com/bitrise-tools/go-xcode/certificateutil"
+
+// FilterCodeSignGroupsForInstallerCertificate ...
+func FilterCodeSignGroupsForInstallerCertificate(codeSignGroups []CodeSignGroup, installedInstallerCertificates []certificateutil.CertificateInfoModel) []CodeSignGroupMac {
+	filteredGroups := []CodeSignGroupMac{}
+
+	for _, group := range codeSignGroups {
+		macGroup := CodeSignGroupMac{
+			CodeSignGroup: group,
+		}
+
+		for _, installerCertificate := range installedInstallerCertificates {
```

Comment:
- collect the possible installer certs for the group, use the first one later (logging)
- create the `CodeSignGroupMac` if it will be saved